### DISCUSSION TITLE
A error which happens when compressing the model

### DISCUSSION THREAD

**Discussion by TerryHangLyu** [deepmodeling/deepmd-kit#1540] [Q&A] [answered]
Dear all,
     I trained a model using the descriptor se_e3 and freeze it into my graph.pb. I tried to further compress it using the command 'dp compress -i frozen_model.pb -o graph-compress.pb'. But some errors happened. But from the manual, seems like that compression works for the descriptor se_e3. How can I deal with this error to compress the model. The compress.json is attache below.
![image](https://user-images.githubusercontent.com/55949849/156884614-af5d9208-99ea-4504-beb4-2acc471464b9.png)
![image](https://user-images.githubusercontent.com/55949849/156884660-3d03fee6-f682-42f0-a9ec-4c7bf630fdd5.png)

Best wishes,
Hang LYU
PS;
compress.json
 {
    "model": {
        "type_map": [
            "Ge",
            "Te"
        ],
        "descriptor": {
            "type": "se_e3",
            "sel": [
                20,
                20
            ],
            "rcut_smth": 0.5,
            "rcut": 5.8,
            "neuron": [
                2,
                4,
                8
            ],
            "resnet_dt": false,
            "seed": 1,
            "activation_function": "tanh",
            "precision": "float64",
            "trainable": true,
            "set_davg_zero": false
        },
        "fitting_net": {
            "neuron": [
                240,
                240,
                240
            ],
            "resnet_dt": true,
            "seed": 1,
            "type": "ener",
            "numb_fparam": 0,
            "numb_aparam": 0,
            "activation_function": "tanh",
            "precision": "float64",
            "trainable": true,
            "rcond": 0.001,
            "atom_ener": []
        },
        "data_stat_nbatch": 10,
        "data_stat_protect": 0.01,
        "compress": {
            "type": "se_e2_a",
            "compress": true,
            "model_file": "graph.pb",
            "min_nbor_dist": 2.3860388183293644,
            "table_config": [
                5,
                0.01,
                0.1,
                -1
            ]
        }
    },
    "learning_rate": {
        "type": "exp",
        "decay_steps": 5000,
        "start_lr": 0.001,
        "stop_lr": 3.51e-08,
        "scale_by_worker": "linear"
    },
    "loss": {
        "start_pref_e": 0.02,
        "limit_pref_e": 1,
        "start_pref_f": 1000,
        "limit_pref_f": 1,
        "start_pref_v": 0,
        "limit_pref_v": 0,
        "type": "ener",
        "start_pref_ae": 0.0,
        "limit_pref_ae": 0.0,
        "start_pref_pf": 0.0,
        "limit_pref_pf": 0.0
    },
    "training": {
        "training_data": {
            "systems": [
                "./data_1"
            ],
            "batch_size": "auto",
            "set_prefix": "set",
            "auto_prob": "prob_sys_size",
            "sys_probs": null
        },
        "validation_data": {
            "systems": [
                "./data_2"
            ],
            "batch_size": "auto",
            "set_prefix": "set",
            "auto_prob": "prob_sys_size",
            "sys_probs": null,
            "numb_btch": 1
        },
        "numb_steps": 1000000,
        "seed": 1,
        "disp_file": "lcurve.out",
        "disp_freq": 10,
        "numb_test": 4,
        "save_freq": 1000,
        "save_ckpt": "model-compression/model.ckpt",
        "disp_training": true,
        "time_training": true,
        "profiling": false,
        "profiling_file": "timeline.json",
        "tensorboard": false,
        "tensorboard_log_dir": "log",
        "tensorboard_freq": 1
    }
}

**Comment by njzjz** [comment, marked as answer]
It's only available in the `devel` branch that hasn't been released yet.

**Reply by TerryHangLyu** [reply]
Thanks for your answer.
